.github/ISSUE_TEMPLATE/Doc_changes.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Documentation Changes
+about: Create a report to help us improve
+
+---
+
+### Document Change needed
+
+A clear description of what changes are needed with the location of the error that needs to be fixed 
```

Comment:
What about something like "A clear description of what changes can be applied to the documentation in order to either make the documentation clearer and easier to follow, or to fix errors in the provided instructions or code."